### PR TITLE
Problem (Fix #850): no automated test for jailing using byzantine faults

### LINKED ISSUE #850: Problem: no automated test for jailing using byzantine faults
two validators at the same time – double sign byzantine fault, one of the validators was “duplicate” (had the same validator key)
3 nodes

@leejw51crypto Jason can help with this issue

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- test procedure:
      - 3 nodes
      - node2 has more than 2/3 voting powers
      - node0 and node1 have same validator seed, both have 0 bonded coins initially, only unbonded coins.
      - start the nodes, and stop node1 immediately.
      - withdraw, deposit and join node0.
      - start node1, check node0 punished with byzantine fault.

### CODE REVIEW

File: integration-tests/multinode/byzantine_test.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+procedure:
+- restore node2 wallet on node0
+- stop node2
+- wait for missed_block_threshold blocks
+- check punishment state on the first staking address
+- start node2
+- wait for node2 port
+- do unjail, expect failure.
+- wait until jail_time + jail_duration
+- do unjail
```
Comment (tomtau) [inline integration-tests/multinode/byzantine_test.py]: this description doesn't seem to match? What is the byzantine fault here?

Comment (yihuang) [reply integration-tests/multinode/byzantine_test.py]: The docstring is copied from jail_test = =".
Updated.

**Comment by yihuang** [comment]
Added the sleep before stop node.

**Pull request merged**